account/src/main/resources/org/killbill/billing/account/ddl.sql

```diff
@@ -23,6 +23,7 @@ CREATE TABLE accounts (
     country varchar(50) DEFAULT NULL,
     postal_code varchar(16) DEFAULT NULL,
     phone varchar(25) DEFAULT NULL,
+    notes varchar(4096) DEFAULT NULL,
```

Comment:
Sure @pierre, do I have to create the SQL file manually with the alter table command?
I've never used the **flyway** commands.

Thanks!